editors/jetbrains/README.md

```diff
@@ -28,3 +28,13 @@ Go to *File | Settings | Tools | External Tools* for Windows and Linux or *WebSt
 Now when you setup **External Tool** I guess you want to add hotkey for it. Go to *File | Settings | Keymap* for Windows and Linux *WebStorm | Preferences | Keymap* and type external tool name in search box.
 
 See [this documentation](https://www.jetbrains.com/help/webstorm/configuring-keyboard-shortcuts.html) about configuring keyboard shortcuts.
+
+## Using File Watcher
+
+To automatically format using `prettier` on save, you can use a file watcher.
+
+Go to *File | Settings | Tools | File Watchers* for Windows and Linux or *WebStorm | Preferences | Tools | File Watchers* for OS X and click **+** to add a new tool. Let’s name it **Prettier**.
+
+* **Program** set the full path to a `prettier` executable, such as `/Users/developer/repo/jest/node_modules/.bin/prettier` (on OS X and Linux) or `C:/\Users\developer\repo\jest\node_modules\.bin\prettier.cmd` (on Windows).
```

Comment:
Relative paths unfortunately doesn't work here... Unsure about the Windows path, I don't have a windows box available to test.